src/components/pages/Navbar/Navbar.js

```diff
@@ -110,7 +110,7 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
               </div>
             )}
 
-            {isAuthenticated && (
+            {!isAuthenticated && (
```

Comment:
This makes sense - showing the buttons if the user is not logged in. Though I'm not sure why it was in this PR, it doesn't seem related to the other changes, and it wasn't mentioned in your video. I guess that's a little bug fix.